std_msgs/Float64 rho_xy_1
std_msgs/Float64 rho_xy_2
std_msgs/Float64 rho_xy_3
std_msgs/Float64 beta_12
std_msgs/Float64 beta_13
std_msgs/Float64 beta_23
